examples/model_selection/plot_grid_search_digits.py

```diff
@@ -27,8 +27,7 @@
 
 # To apply an classifier on this data, we need to flatten the image, to
 # turn the data in a (samples, feature) matrix:
-n_samples = len(digits.images)
-X = digits.images.reshape((n_samples, -1))
+X = digits.data
```

Comment:
The comment above this line needs to be adjusted since we do not need to flatten ourselves anymore.